cfe_internal/enterprise/CFE_knowledge.cf

```diff
@@ -51,17 +51,43 @@ bundle agent cfe_internal_setup_knowledge
       "$(install_logs)" -> { "ENT-4506" }
         perms => mog("0600", "root", "root" );
 
-      "$(cfe_internal_hub_vars.docroot)"
-      comment => "All files in there should be at least 0460. Giving cfapache group full access",
-      handle => "cfe_internal_setup_knowledge_files_doc_root_2",
-      file_select => cfe_internal_docroot_perms,
-      perms => mog("0460", "root", $(def.cf_apache_group) ),
-      depth_search => recurse_exclude("inf");   # see exclude dirs in recurse_exclude() body
-
       "$(cfe_internal_hub_vars.docroot)/."
-      comment => "The top level docroot dir should be at least 0460 with root:cfapache",
+      comment => "The top level docroot needs to be readable and executable by the web server.",
       handle => "cfe_internal_setup_knowledge_dir_doc_root",
-      perms => mog("0460", "root", $(def.cf_apache_group) );
+      perms => mog("0450", "root", $(def.cf_apache_group) );
```

Comment:
It doesn't make much sense to give the *root* user read access but not execute. Strictly speaking this would mean even the agent process could not access and modify files in these directories.

Reply:
ACK, mog( 0550 )